doc/dev/toolchain-setup.md

```diff
@@ -79,3 +79,8 @@ Note the checkout and build will take some time, and the build process may have
 This should put the binaries in a `bin` subdirectory of your build folder (the `builds/cheriot-llvm` folder you made).
 
 With the CHERIoT/LLVM toolchain built, you can now continue to setup the examples and build them.
+
+### Additional Sonata Software dependencies
+
+To set-up a manual environment that can build Sonata Software and RTOS you can follow the [building dependencies section of the RTOS guide](https://github.com/CHERIoT-Platform/cheriot-rtos/blob/main/docs/GettingStarted.md#building-the-dependencies).
```

Comment:
Grammar nit: "set-up" is an alternative form of the noun form of "setup", whereas here it should be the verb form "set up".

Also might be worth writing "... Software and **CHERIoT** RTOS ..." just to remove doubt about which RTOS you mean, since I don't think it's mentioned elsewhere in this markdown file.

Reply:
Good point, these should now be changed.